src/firefly/js/ui/ColorPicker.jsx

```diff
@@ -16,12 +17,16 @@ import {getRGBA} from 'firefly/util/Color.js';
 const DEF_PRESET_COLORS= ['#D0021B', '#F5A623', '#F8E71C', '#8B572A', '#7ED321', '#417505', '#BD10E0',
     '#9013FE', '#4A90E2', '#50E3C2', '#B8E986', '#000000', '#4A4A4A', '#9B9B9B', '#FFFFFF'];
 
-const dialogTip=
-`Choose a color:
-- graphically
-- as an RGB hex color code (does not support alpha)
-- by entering RGB values (0-255) and an alpha in percent (0-100)
-- from the swatches at the bottom`;
+const dialogTip= (
+    <Typography sx={{'ul li': {listStyleType:'circle'}}}>
```

Comment:
You can get away with only `li` if you want to since it applies to only this block.